_posts/2020-06-10-experiences-using-liberating-structures.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title:  "Experiences of using liberating structures"
+date:   2020-06-10 13:20:00
+categories: liberating structures experiences
+---
+
+
+Liberating structures promise to unleash innovation, creativity and great ideas. I capture my thoughts and feelings, from my experiences with liberating  microstructures. 
+
+As part of this post I will include a brief overview of what these microstructures are, for more information about specific structures see the [liberating structure website][liberatingstructures].
+
+
+[liberatingstructures]: http://www.liberatingstructures.com/
+
+## What are Structures
+
+Structures are the various parts that come together to make a workplace/enterprise. These are split into both 'Macrostructures', 'Microstructures' and 'Structural elements'.
+* Macrostructures are the larger and slower moving items such as the core operating model, buildings and strategy.
+* Microstructures are the smaller parts such as; meeting rooms, brainstorm and discussions. This is where and how discussions take place.
+* Structural elements facilitate the microstructures. For example, whiteboards, time allocations and agendas.
+
+
+All the structures play an important role, as each builds upon the smaller elements. Small changes on the lower levels over time can significantly effect the bigger structures. I have noticed for example, that virtual meetings seem to have a smaller pool of ideas and converge to a single idea much faster compared to in person meetings. Possibly, due to the fact virtual meetings only have one person speaking and it can be difficult to edge in. This reduces the pool of ideas and the best idea may not be chosen.
+
+It's important to note that each level of the structures listed above include both tangible and intangible elements. Liberating structures are specifically different ways to have a meeting/discussion/report. This puts them in the category of intangible microstructures.
+
+## Conventional Structures
+
+There are five main conventional structures that people use:
+* Presentations
+* Status Reports
+* Managed Discussions
+* Open Discussions
+* Brainstorms
+
+These are just the default options for information sharing, however they have a few problems as the first three are inhibiting as they don't allow everyone to get involved and often restrict the majority of the talking to a small amount of people. The final two (open discussions and brainstorms) are often too loose and disorganised to drive actual outcomes. 
```

Comment:
Suggested rewording:

"These are the most common options for information sharing, but they aren't without their problems. The first three structures are inhibiting as they don't allow everyone to get involved and often restrict the majority of the talking to a small amount of people. On the other hand, the final two are often too loose and disorganised to drive actual outcomes."

I'd suggest maybe including a linking sentence like "However, these problems can be solved through the use of liberating structures."